docs/README.md

```diff
@@ -2,7 +2,7 @@
 
 These are Sphinx docs, automatically built at [https://docs.wagtail.org](https://docs.wagtail.org) when the `main` branch is committed to GitHub. To build them locally, install Wagtail's development requirements (in the root Wagtail directory):
 
-    pip install -e .[testing,docs]
+    pip install -e .[testing,docs] --config-settings editable-mode=strict
```

Comment:
Note that this isn't really related to this PR. In my experience, VSCode won't be able to pick up imports without this option set explicitly to either `strict` or `compat`, both before and after this PR.

I'm not sure if we want this in the docs. Could you please test whether VSCode can pick up `wagtail` imports if you run `pip install -e path/to/wagtail` without `--config-settings editable-mode`, both before and after this PR?

If it works for you, maybe it's just something on my end.

If it doesn't, and if we don't want to add this in the docs, we can also look into other build backends which supposedly doesn't have this problem.

Reply:
I can confirm I'm seeing the same behaviour - both before and after this PR, VSCode does not recognise `wagtail` imports when wagtail has been installed with `pip install -e path/to/wagtail`, and _does_ recognise them when installed with `pip install -e path/to/wagtail --config-settings editable-mode`. Happy to go ahead with this change.